apps/app_rpt.c

```diff
@@ -3226,7 +3226,7 @@ static inline void periodic_process_links(struct rpt *myrpt, const int elap)
 		if ((!l->chan) && (!l->retrytimer) && l->outbound && (l->retries++ < l->max_retries) && (l->hasconnected)) {
 			if (l->chan)
 				ast_hangup(l->chan);
-			l->chan = 0;
+			l->chan = NULL;
```

Comment:
I was going to ask about this - there are mixed instances where `l->chan = 0` and others where `l->chan = NULL`.  Maybe we should make them consistent.  0 != NULL , but I believe if tests give the same result.